clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -226,7 +231,38 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
       }
     }
 
-    def metadata = GCEUtil.buildMetadataFromMap(description.instanceMetadata)
+    // Update the instance metadata for ILBs and queue up region backend service calls.
+    List<BackendService> regionBackendServicesToUpdate = []
+    if (internalLoadBalancers) {
+      List<String> existingRegionalLbs = instanceMetadata[GoogleServerGroup.View.REGIONAL_LOAD_BALANCER_NAMES]?.split(",") ?: []
+      def ilbServices = internalLoadBalancers.collect { it.backendService.name }
+      def ilbNames = internalLoadBalancers.collect { it.name }
+
+      ilbNames.each { String ilbName ->
+        if (!(ilbName in existingRegionalLbs))  {
+          existingRegionalLbs << ilbName
+        }
+      }
+      instanceMetadata[GoogleServerGroup.View.REGIONAL_LOAD_BALANCER_NAMES] = existingRegionalLbs.join(",")
+
+      ilbServices.each { String backendServiceName ->
+        BackendService backendService = compute.regionBackendServices().get(project, region, backendServiceName).execute()
```

Comment:
`GET`s aren't as prone to retry-able failures as `UPDATE`s are. The Http(s) LBs don't use retries here, so I won't here until it becomes an issue.